plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestingKuduServer.java

```diff
@@ -34,7 +34,7 @@
 public class TestingKuduServer
         implements Closeable
 {
-    private static final String KUDU_IMAGE = "apache/kudu:1.10.0";
+    private static final String KUDU_IMAGE = "apache/kudu:1.15.0";
```

Comment:
Can it be a part of the first commit ?

Reply:
Sure, I wanted to verify the kudu 1.15 client could also talk to kudu 1.10 server. The docs say the wire protocol is backwards compatible, but wanted to verify.

But I can move this into one commit now.